### DISCUSSION TITLE
Reapply tags to all documents

### DISCUSSION THREAD

**Discussion by pawamoy** [jonaswinkler/paperless-ng#1244] [Support] [answered]
If I create a new tag with a matching algorithm and pattern, can I auto-apply it to all matching documents? I couldn't find any button to do such a thing.

**Comment by Pyromane** [comment]
There is a need for this!
See also: https://github.com/jonaswinkler/paperless-ng/discussions/1033

**Comment by torbengb** [comment, marked as answer]
Do you know the "document retagger" management function? It sounds like it would do what you ask:
https://paperless-ng.readthedocs.io/en/latest/administration.html#document-retagger

**Reply by pawamoy** [reply]
Indeed, seems like the answer :slightly_smiling_face: 
Thanks!
